code/Routing/DateLinkController.php

```diff
@@ -24,6 +24,10 @@ public function getNestedController() {
 				"URLSegment" => $URLSegment,
 				"ParentID" => $parentID
 			))->first();
+		
+		if( true !== $sitetree instanceof SiteTree ){
```

Comment:
Can you please use `if ( ! $sitetree instanceof SiteTree)` instead of a bool comparison?